app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaTest.kt

```diff
@@ -338,6 +338,42 @@ class CtaTest {
         assertEquals("<b>Facebook</b>withZero", value)
     }
 
+    @Test
```

Comment:
Random opinion: I don't think these tests are giving us any value. Feel free to remove them if you want.

Reply:
👍